### PR TITLE
Add empty checks to Utils

### PR DESCRIPTION
**Pull request by dhawal1248**

### CODE REVIEW

File: v1/util/index.js [hunk 1]
```diff
@@ -353,7 +355,9 @@ function flattenJson(data) {
       if (isEmpty && prop) result[prop] = {};
     }
   }
-  recurse(data, "");
+  if (data) {
```
Comment (sayan-mitra) [inline v1/util/index.js:358]: should we add the validation for primitive types as well, like 
flattenJson("abc")/ flattenJson([1,2,3]) etc?

File: v1/util/index.js [hunk 2]
```diff
@@ -42,9 +42,11 @@ const formatValue = value => {
 // Format the destination.Config.dynamicMap arrays to hashMap
 const getHashFromArray = (arrays, fromKey = "from", toKey = "to") => {
   const hashMap = {};
-  arrays.forEach(array => {
-    hashMap[array[fromKey]] = array[toKey];
-  });
+  if (arrays) {
```
Comment (sayan-mitra) [inline v1/util/index.js:45]: let's do Array.isArray(arrays) here?
it wlll then safeguard objects too

**Review by sayan-mitra — CHANGES_REQUESTED** [review]

File: v1/util/index.js (hunk 1 above)
Comment (dhawal1248) [reply v1/util/index.js:362]: Done

File: v1/util/index.js (hunk 2 above)
Comment (dhawal1248) [reply v1/util/index.js]: Done

**Review by sayan-mitra — APPROVED** [review]

**Pull request merged**